slang.h

```diff
@@ -695,6 +710,8 @@ extern "C"
     SLANG_API SlangUInt spReflection_getEntryPointCount(SlangReflection* reflection);
 
     SLANG_API SlangReflectionEntryPoint* spReflection_getEntryPointByIndex(SlangReflection* reflection, SlangUInt index);
+    SLANG_API SlangUInt spReflection_getGlobalConstantBufferBinding(SlangReflection* reflection);
```

Comment:
This seems fine, and is worth keeping as a convenience API, but I would have liked to see something that just returns a `SlangReflectionVarLayout` for the global scope.